### ISSUE TITLE
Plugin not doing anything

### ISSUE THREAD

**Issue by DeflatedPickle** [hendraanggrian/packr-gradle-plugin#9]
I setup [a small example](https://github.com/DeflatedPickle/packr-gradle-test) using the plugin after testing it on my main project. In both cases, when I run `packAll`, no files are created

The output I get is the following;
```
12:53:04: Executing task 'packAll'...

> Task :packLinux32
> Task :packLinux64
> Task :packMacOS
> Task :packWindows32
> Task :packWindows64
> Task :packAll

BUILD SUCCESSFUL in 133ms
5 actionable tasks: 5 executed
12:53:05: Task execution finished 'packAll'.
```
But the program isn't bundled at the output I specified

This could be because I don't have any `tasks` configured, but when I try to follow the example `tasks`, I get the following (on the same project as above);
```
> Could not get unknown property 'executable' for task ':packWindows64' of type com.hendraanggrian.packr.PackTask.
```
A similar error occurs for any property attempt

**Comment by hendraanggrian** [comment]
`com.hendraanggrian.packr` is the id I used when this plugin is hosted in jcenter, which is deprecated by now. Since then, this plugin has been uploaded to [Gradle Plugin Portal](https://plugins.gradle.org/plugin/com.hendraanggrian.packaging) using id `com.hendraanggrian.packaging`. I'm surprised you can even download `com.hendraanggrian.packr`.

Anyway, please use `com.hendraanggrian.packaging` and see how it works.

**Comment by DeflatedPickle** [comment]
That's my fault, you do mention it in the README. Sorry about that! I did originally try it as detailed in the README, with the plugins DSL. But that was throwing an error. However, I tried it with the legacy method of applying plugins and it worked, given I changed to using a `packaging` block and `set` methods. Though in my case, it's still not working as intended. I get up to the `Unpacking JRE ...` stage before being given the following error;
```
Execution failed for task ':packLinux64'.
> java.nio.file.NoSuchFileException: /home/plight/.jdks/adopt-openjdk-1.8.0_292/jre/lib/amd64/server/libjsig.debuginfo
```
I have no idea why it's trying to use this file and I can't find anything about acquiring this file on Ubuntu, but I will comment again if I figure it out!
